compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -1406,6 +1398,21 @@ private void handleDeclaredWithVar(BLangVariable variable) {
         }
     }
 
+    private void handleWildCardBindingVariable(BLangSimpleVariable simpleVariable, BType rhsType) {
+        if (simpleVariable.name.value.equals(Names.IGNORE.value)) {
```

Comment:
Do we need to consider `Names.EMPTY` case as well?

Reply:
Any idea when it can be empty? As we are setting ignore name at node transformer for wildcard binding https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java#L2433

Reply:
Yes, `BLangNodeTransformer` doesn't set `Names.EMPTY` for anything. Also, It sets `missingnode` for incomplete quoted identifier (`'`) as well. But we have a check like this in `SymbolEnter` for `BLangSimpleVariable` https://github.com/ballerina-platform/ballerina-lang/blob/37a2e6c47008900d03b53d28479d9714477796d6/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java#L2035
Should we update this as well if not needed?

Reply:
When we do `symbolEnter.define()` say from desugar; function returns, params might need to be filtered. eg: `function foo returns int`